activerecord/lib/active_record/core.rb

```diff
@@ -129,6 +129,14 @@ def self.configurations
       # for multiple databases.
       mattr_accessor :suppress_multiple_database_warning, instance_writer: false, default: false
 
+      ##
+      # :singleton-method:
+      # Force enumeration of all columns in SELECT statements
+      # e.g. `SELECT first_name, last_name FROM ...` instead of `SELECT * FROM ...`
+      # This avoids PreparedStatementCacheExpired errors when a column is added
+      # to a Postgres database while the app is running.
+      mattr_accessor :enumerate_columns_in_select_statements, instance_accessor: false, default: false
```

Comment:
This should be a `class_attribute` to allow models to override it per-model basis.

Reply:
Updated!